cg/meta/workflow/analysis.py

```diff
@@ -609,26 +619,49 @@ def _write_managed_variants(out_dir: Path, content: list[str]) -> None:
 
     @staticmethod
     def _write_panel(out_dir: Path, content: list[str]) -> None:
-        """Write the managed variants to case dir."""
+        """Write the gene panel to case dir."""
         out_dir.mkdir(parents=True, exist_ok=True)
         WriteFile.write_file_from_content(
             content="\n".join(content),
             file_format=FileFormat.TXT,
             file_path=Path(out_dir, ScoutExportFileName.PANELS),
         )
 
-    def _get_gene_panel(self, case_id: str, genome_build: str) -> list[str]:
+    def _get_gene_panel(self, case_id: str, genome_build: str, dry_run: bool = False) -> list[str]:
         """Create and return the aggregated gene panel file."""
         case: Case = self.status_db.get_case_by_internal_id(internal_id=case_id)
         all_panels: list[str] = self.get_aggregated_panels(
             customer_id=case.customer.internal_id, default_panels=set(case.panels)
         )
-        return self.scout_api.export_panels(build=genome_build, panels=all_panels)
+        return self.scout_api.export_panels(build=genome_build, panels=all_panels, dry_run=dry_run)
+
+    def get_gene_panel(self, case_id: str, dry_run: bool = False) -> list[str]:
```

Comment:
Could you add the `dry_run` option to the child methods (mip rna & mip dna) 🙏

Reply:
and of course if I change this these pytests fail and need to be adapted. I'm giving up today. I will see tomorrow if I decide to address it or not

Reply:
Mmmm they should not, It's adding the argument to the child methods with a default value of false and passing it to the _get_gene_panel method

Reply:
sorry I misunderstood your comment. Now I get it. Done